src/__tests__/profiling-test.js

```diff
@@ -16,17 +16,19 @@ describe('profiling', () => {
 
   const exportImportHelper = (rendererID: number, rootID: number) => {
     const {
-      prepareProfilingExport,
-      prepareProfilingImport,
+      prepareExportedProfilingSummary,
+      prepareImportedProfilingData,
     } = require('src/devtools/views/Profiler/utils');
 
-    let exportedProfilingSummary;
+    let exportedProfilingDataJsonString = '';
     bridge.addListener('exportFile', ({ contents }) => {
-      exportedProfilingSummary = contents;
+      if (typeof contents === 'string') {
+        exportedProfilingDataJsonString = (contents: string);
+      }
     });
 
     utils.act(() => {
-      const exportProfilingSummary = prepareProfilingExport(
+      const exportProfilingSummary = prepareExportedProfilingSummary(
         store.profilingOperations,
         store.profilingSnapshots,
         rootID,
```

Comment:
There's a bug hiding here because we do not use the help that Flow opaque types can provide:
```
        const exportProfilingSummary = prepareExportedProfilingSummary(
          store.profilingOperations,
          store.profilingSnapshots,
          rootID,
          rendererID
        );
```
```
export const prepareExportedProfilingSummary = (
  profilingOperations: ImportedProfilingDataProfilingOperations,
  profilingSnapshots: ImportedProfilingDataProfilingSnapshots,
  rendererID: number,
  rootID: number
) => {
```
All numbers are the same, but not really.